services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue

```diff
@@ -1,40 +1,84 @@
+<!-- Used to display presentation received after RFC-0037 Present-Proof-v1.0
+
+ https://github.com/hyperledger/aries-rfcs/blob/main/features/0037-present-proof/README.md
+ 
+ May work for RFC-0454 Present-Proof-v2.0, to be tested later.... and update this comment
+
+ https://github.com/hyperledger/aries-rfcs/blob/main/features/0454-present-proof-v2/README.md
+ -->
+
 <template>
   <div v-if="presentation">
     <ul>
-      <li>Status: {{ presentation.status }}</li>
-      <li>Updated at: {{ formatDateLong(presentation.updated_at) }}</li>
-      <li>Contact Alias: {{ presentation.contact.alias }}</li>
-      <hr />
-      <li
-        v-for="(value, attr, index) in props.presentation.acapy
-          .presentation_exchange.presentation.requested_proof
-          .revealed_attr_groups"
-        :key="index"
-      >
-        <h4>{{ attr }}:</h4>
-        <span
-          v-for="(val, attr_name, i) in value.values"
+      <div v-if="props.header">
+        <li>Status: {{ presentation.status }}</li>
+        <li>Updated at: {{ formatDateLong(presentation.updated_at) }}</li>
+        <li>Contact Alias: {{ presentation.contact.alias }}</li>
+        <hr />
+      </div>
+      <div v-if="presentation.acapy.presentation_exchange.presentation">
+        <!-- PRESENTATION RECEIVED-->
+        <!-- requested_attributes using 'names' list -> revealed attribute groups -->
+        <li
+          v-for="(value, attr, index) in props.presentation.acapy
+            .presentation_exchange.presentation.requested_proof
+            .revealed_attr_groups"
+          :key="index"
+        >
+          <h4>{{ attr }}:</h4>
+          <span
+            v-for="(val, attr_name, i) in value.values"
+            :key="i"
+            class="presentation-attr-value"
+          >
+            <b>{{ attr_name }}</b> : {{ val.raw }}
+            <br />
+          </span>
+        </li>
+        <!-- requested_attributes using 'name' string w/ restrictions -> revealed attributes -->
+        <li
+          v-for="(val, attr_name, i) in presentation.acapy.presentation_exchange
+            .presentation.requested_proof.revealed_attrs"
           :key="i"
           class="presentation-attr-value"
         >
-          <b>{{ attr_name }}</b> : {{ val.raw }}
-          <br />
-        </span>
-      </li>
+          <b>{{
```

Comment:
Generally use `<strong>` over bold tag in html5 these days
https://html.com/tags/b/